README.md

```diff
@@ -277,36 +299,6 @@ Python Interactive Interpreter. To exit, type 'quit()', 'exit()' or Ctrl-D.
 It's recommended that you use the latest LLDB release from the official [APT repo](https://apt.llvm.org/) instead
 of the one shipped with your distro.
 
-Conflicting Python versions on Mac OS
--------------------------------------
-
-If you see an error like this:
-
-```
-Traceback (most recent call last):
-  File "<input>", line 1, in <module>
-  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/io.py", line 51, in <module>
-    import _io
-ImportError: dlopen(/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so, 2): Symbol not found: __PyCodecInfo_GetIncrementalDecoder
-  Referenced from: /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so
-  Expected in: flat namespace
- in /usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload/_io.so
-```
-
-then the version of LLDB, that is shipped with Mac OS and linked against the system CPython,
-is trying to use CPython installed via Homebrew. This won't work. You need to make sure LLDB
-picks up the correct CPython version on start. One way to achieve that would be modifying
-`$PATH`, e.g. by creating a wrapper for `lldb`:
```

Comment:
In my case LLDB never cared about PATH in context of CPython plugins and REPL. I'd suppose this might have worked with LD_LIBRARY_PATH, but I have not tried.